assets/xml/objects/object_gi_arrow.xml

```diff
@@ -1,7 +1,8 @@
 ﻿<Root>
+    <ExternalFile XmlPath="objects/gameplay_keep.xml" OutPath="assets/objects/gameplay_keep/"/>
     <File Name="object_gi_arrow" Segment="6">
```

Comment:
I hate to be "that guy", but can you add a description of the asset above the <File> node for all of these? Maybe above the <ExternalFile> node too. You can just copy what's in the spreadsheet for now.